examples/requests.jsonl

```diff
@@ -12,3 +12,4 @@
 {"id":"324cbaba6d05a483a5501925daa0baf0fdf5cd204aac0af08c86ba661afeef2f", "kind":"ContainerImage","resource":"quay.io/jetstack/cert-manager-controller:v1.16.0"}
 {"kind": "GitRepo", "id": "low-priority", "resource": "https://github.com/leaktk/fake-leaks", "options": {"priority": 0}}
 {"kind": "GitRepo", "id": "high-priority", "resource": "https://github.com/leaktk/fake-leaks", "options": {"priority": 1}}
+{"id":"580efb11ee9013a42d6037566c7159c9aeb610696be851dc9209c85e75e5a3e7", "kind":"JSONData", "resource":"{\"url1\":\"https://notreal.co.asd\", \"url2\":\"http://www.google.com\", \"some\":{\"key\": \"-----BEGIN PRIVATE KEY-----c5602d28d0f21422dfc7b572b17e6b138c1b49fd7f477d4c5c961e0756f1ff70-----END PRIVATE KEY-----\"}}", "options": {"fetch_urls":  true}}
```

Comment:
May want to change fetch_urls to `"**"` after https://github.com/leaktk/scanner/pull/66 is merged in.